packages/manager/apps/web/client/app/hosting/popover/multisite.html

```diff
@@ -8,6 +8,15 @@
     >
         <span data-translate="hosting_tab_DOMAINS_table_popover_modify"></span>
     </oui-button>
+    <oui-button
+        data-variant="link"
+        class="d-block"
```

Comment:
Not already the case ?

Reply:
i dont understand, can you give more info please

Reply:
Isn't `oui-button` already having a `display:block`

Reply:
i think no it should be added to be block.

Reply:
non it's inline-block by default